.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "indent": [2, 4, {"SwitchCase": 1}],
```

Comment:
There's plenty more settings that could/should be added, but that would make this PR even bigger, so just indention is more than sufficient for this PR.